### DISCUSSION TITLE
Keep track of which libraries export config, for examples in documentation

### DISCUSSION THREAD

**Discussion by borkdude** [clj-kondo/clj-kondo#1528] [General]
See: 

https://github.com/clj-kondo/clj-kondo/blob/master/doc/config.md#exporting-and-importing-configuration

- https://github.com/clj-commons/rewrite-clj/tree/main/resources/clj-kondo.exports/rewrite-clj/rewrite-clj
- https://github.com/nubank/state-flow/tree/master/resources/clj-kondo.exports/nubank/state-flow
- https://github.com/marick/Midje/tree/master/test-resources/clj-kondo.exports/marick/midje
- https://github.com/fulcrologic/fulcro/tree/develop/src/clj-kondo/clj-kondo.exports/com.fulcrologic/fulcro
- https://github.com/lilactown/helix/tree/master/resources/clj-kondo.exports/lilactown/helix
- https://github.com/wilkerlucio/pathom3/tree/master/resources/clj-kondo.exports/com.wsscode/pathom3
- https://github.com/CrypticButter/snoop/tree/main/resources/clj-kondo.exports/com.crypticbutter/snoop
- https://github.com/lread/status-line/tree/main/resources/clj-kondo.exports/lread/status-line
- https://github.com/seancorfield/next-jdbc/tree/develop/resources/clj-kondo.exports
- https://github.com/clojure-expectations/clojure-test/tree/develop/resources/clj-kondo.exports/com.github.seancorfield/expectations
- https://github.com/holyjak/clj-concordion/tree/master/src/clj-kondo/clj-kondo.exports/clj-concordion/clj-concordion
- https://github.com/Engelberg/better-cond/tree/master/resources/clj-kondo.exports/better-cond/better-cond
- https://github.com/johnmn3/injest/tree/main/src/clj-kondo/clj-kondo.exports/net.clojars.john/injest
- https://github.com/metosin/malli/tree/master/resources/clj-kondo/clj-kondo.exports/metosin/malli
- https://github.com/sicmutils/sicmutils/tree/main/resources/clj-kondo.exports/sicmutils/sicmutils
- https://github.com/funcool/promesa/blob/master/resources/clj-kondo.exports/promesa/config.edn

**Comment by ericdallo** [comment]
https://github.com/marick/Midje/tree/master/test-resources

**Comment by ghostandthemachine** [comment]
https://github.com/lilactown/helix/tree/master/resources/clj-kondo.exports/lilactown/helix

**Comment by lread** [comment]
https://github.com/lread/status-line/tree/main/resources/clj-kondo.exports/lread/status-line

**Comment by ikitommi** [comment]
https://github.com/metosin/malli/tree/master/resources/clj-kondo/clj-kondo.exports/metosin/malli

**Comment by sritchie** [comment]
https://github.com/sicmutils/sicmutils/tree/main/resources/clj-kondo.exports/sicmutils/sicmutils

Installation instructions and descriptions of the custom linter warnings I emit live here: https://github.com/sicmutils/sicmutils/blob/main/doc/linters.md

**Comment by marksto** [comment]
Community-maintained hooks for `encore`:
https://github.com/taoensso/encore/tree/master/resources/clj-kondo.exports/taoensso/encore
